src/components/Table/SimpleTable.js

```diff
@@ -125,13 +143,15 @@ class SimpleTable extends React.Component {
 
   renderRowDetails = () => {
     const itemIdx = this.indexOfDetailsItem();
-    if (!this.props.renderDetails || !this.state.detailsItem || itemIdx === -1) {
+    const item = this.props.items[itemIdx];
```

Comment:
Move this line below the conditional check; we don't need `item` until we're sure we will render it.